pyproject.toml

```diff
@@ -29,6 +29,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
+xarray = '>=2023.0.0'
```

Comment:
xarray bring in pandas as a dependency, but if you are explicitly importing pandas it'd be good to add that to the optional test dependencies as well.